server/parameters/CoreConfig.java

```diff
@@ -191,10 +191,15 @@ public static class File extends Type {
                     private final YAMLParser.Value.TimePeriodName archiveGrouping;
                     private final YAMLParser.Value.TimePeriod archiveAgeLimit;
                     private final long archivesSizeLimit;
+                    private final String filename;
+                    private final String extension;
 
-                    File(Path baseDirectory, long fileSizeLimit, YAMLParser.Value.TimePeriodName archiveGrouping,
+                    File(Path baseDirectory, String filename, String extension,
```

Comment:
So the config output has two new paramters, `filename` and `extension`?
Does this mean we've changed the config yaml file?